### PR TITLE
Fixed page reload issue and date time picker render

### PR DESCRIPTION
**Pull request by arunapa**
Issue: 1) DateTimePicker widget type does not render correctly
2) After submitting the metadata definition, empty page is loaded

Fix: 1) Migrate DateTimePicker away from `@mui/lab`, added Dayjs package as per MUI development guide

Screen Shot 2022-11-21 at 10 55 52 AM

### CODE REVIEW

**Comment by ddey2** [comment]
I tried adding metadata definition of type 'dateTimePicker'. It doesn't get created. I see this error on backend. 

Screen Shot 2022-11-21 at 11 11 20 AM

**Review by ddey2 — CHANGES_REQUESTED** [review]
metadata of type 'datetimepicker' is not created

File: frontend/package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
 		"@mui/lab": "^5.0.0-alpha.73",
```
Comment (longshuicy) [inline frontend/package.json:41]: I think we can safely remove mui/lab since it's not used anywhere now

**Comment by lmarini** [comment]
Datetime picker worked for me. Maybe try `npm install`?

**Comment by longshuicy** [comment]
I can create an definition of date time; but somehow i cannot update it
```
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/applications.py", line 124, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/middleware/errors.py", line 184, in __call__
    raise exc
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/middleware/errors.py", line 162, in __call__
    await self.app(scope, receive, _send)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/middleware/cors.py", line 92, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/middleware/cors.py", line 147, in simple_response
    await self.app(scope, receive, send)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 75, in __call__
    raise exc
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 64, in __call__
    await self.app(scope, receive, sender)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/routing.py", line 680, in __call__
    await route.handle(scope, receive, send)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/routing.py", line 275, in handle
    await self.app(scope, receive, send)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/starlette/routing.py", line 65, in app
    response = await func(request)
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/fastapi/routing.py", line 231, in app
    raw_response = await run_endpoint_function(
  File "/Users/cwang138/.local/share/virtualenvs/backend-pkZ6g-7g/lib/python3.9/site-packages/fastapi/routing.py", line 160, in run_endpoint_function
    return await dependant.call(**values)
  File "/Users/cwang138/Documents/Clowder/clowder2/backend/./app/routers/metadata_datasets.py", line 112, in add_dataset_metadata
    md = await _build_metadata_db_obj(db, metadata_in, dataset, user)
  File "/Users/cwang138/Documents/Clowder/clowder2/backend/./app/routers/metadata_datasets.py", line 44, in _build_metadata_db_obj
    contents = await validate_context(
  File "/Users/cwang138/Documents/Clowder/clowder2/backend/./app/models/metadata.py", line 281, in validate_context
    contents = validate_definition(contents, md_def)
  File "/Users/cwang138/Documents/Clowder/clowder2/backend/./app/models/metadata.py", line 142, in validate_definition
    contents[field.name] = t(contents[field.name])
TypeError: descriptor 'time' for 'datetime.datetime' objects doesn't apply to a 'str' object
```

**Comment by longshuicy** [comment]
in `metadata.config.tsx` i think 
```
"date": "Date",
    "time": "Time",
```
 should just be 
 ```
 "str": "Date",
"str": "Time"
```
because python basic type only support `str`, `float`, `int`, `bool`, `enum` and `tuple`?

**Review by lmarini — APPROVED** [review]

**Review by longshuicy — APPROVED** [review]

**Review by ddey2 — APPROVED** [review]
Looks good.

**Pull request merged**